web/datafed-ws.js

```diff
@@ -66,7 +66,8 @@ var g_host,
     g_ver_api_major,
     g_ver_api_minor,
     g_ver_api_patch,
-    g_tls;
+    g_tls,
+    g_ga;
```

Comment:
I prefer more descriptive names then ga, maybe goog_analy, or google_analytics. My approach is if I can open a file and know what something is it is a good name. ga is a bit ambiguous.